lldb/tools/lldb-dap/DAP.h

```diff
@@ -383,6 +383,12 @@ struct DAP {
 
   InstructionBreakpoint *GetInstructionBPFromStopReason(lldb::SBThread &thread);
 
+  FunctionBreakpoint *GetFunctionBPFromStopReason(lldb::SBThread &thread);
+
+  FunctionBreakpoint *GetFunctionBreakPoint(const lldb::break_id_t bp_id);
+
+  void WaitWorkerThreadsToExit();
```

Comment:
What is this?

Reply:
Removed it, it got added when resolving the merge conflict.